src/components/Output/index.tsx

```diff
@@ -127,6 +216,41 @@ export default class Output extends Component<Props, State> {
             />
           </pinch-zoom>
         </two-up>
+
+        <div class={style.controls}>
+          <div class={style.group}>
+            <button class={style.button} onClick={this.zoomOut}>
+              <span class={style.icon}>−</span>
+            </button>
+            {editingScale ? (
+              <input
+                type="number"
+                step="1"
+                min="1"
+                max="1000000"
+                ref={linkRef(this, 'scaleInput')}
+                class={style.zoom}
+                value={Math.round(scale * 100)}
+                onInput={this.setScale}
```

Comment:
I played with various implementations, and onChange is particularly poor because of the focus prevention we have in <pinch-zoom> right now. You have to find one of the tiny clickable areas outside the image in order to blur the input and commit the value.

Reply:
(Also IMO much of the weirdness come from the centering issue)

Reply:
Hmm, I'm unsure about this, but we can get Mustafa take a look after this lands (and maybe look at what other apps do).